homeassistant/components/binary_sensor/knx.py

```diff
@@ -107,7 +107,6 @@ def __init__(self, hass, device):
         """Initialize of KNX binary sensor."""
         self.device = device
         self.hass = hass
-        self.async_register_callbacks()
         self.automations = []
```

Comment:
Why is this stored on the entity? It doesn't seem to be used.

Reply:
it is added from outside on line 97

Reply:
And where do we access `entity.automations`?

Reply:
It is not accessed, but we need to store them somewhere.

Reply:
Ok.